### ISSUE TITLE
About scatter points annotation

### ISSUE THREAD

**Issue by hetheclif** [gmplot/gmplot#64]
Hi, I want to know how can I add an annotation to every point? Can anyone help me?
Thanks

**Comment by vgm64** [comment]
I believe there is an issue raised or a pull request that has text labels
on points. The feature doesn't current exist.

Michael Woods

**Comment by frslm** [comment]
@hetheclif, this feature was just merged into master in #152 - you can now specify a title or label for each marker using `scatter()`:
```python
map.scatter(lats, lngs, marker=True,
            title=['First', 'Second', 'Third', 'Fourth'],
            label=['A', 'B', 'C', 'D']
)
```
